vignettes/submodel_documentation_overview.Rmd

```diff
@@ -0,0 +1,394 @@
+---
+title: "Fall Run Submodel Overview"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Fall Run Submodel Overview}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+library(tidyverse)
+library(plotly)
+
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This document describes the variables that contribute to each submodel within the Reorienting to Recovery Decision Support Model. When applicable, it also lists literature that contributed to the development of each submodel. 
+
+## Adult  
+
+### Adult Removal For Hatchery 
+
+On tributaries with existing hatcheries, a percent of natural adults will be removed annually to be used in hatcheries. This is represented in the model by a natural adult removal rate that is applied to adult populations after they return to tributaries. In the model, this information is contained in the `natural_adult_removal_rate` data object and it is input to the `fall_run_model` as part of the params list. 
```

Comment:
do we want to specify what "used" means here? i.e. "a percent of natural adults will be removed from the spawning population to be processed in hatcheries" or something along those lines

Reply:
updated to "removed annually from in river spawning grounds to be spawned in hatcheries"